capstone/home.html

```diff
@@ -0,0 +1,134 @@
+      <div class="holder">
+        <h2 class="title">Main Program</h2>
+        <div class="underLine"></div>
+      </div>
+      <article class="programElements">
+        <article class="elements">
+          <i class="fa-sharp fa-solid fa-hourglass-start"></i>
+          <h3>Tour starts</h3>
+          <p>It's a long way to the top if you want to rock & roll, sometimes you have to start at the bottom. </p>
+        </article>
+        <article class="elements">
+          <i class="fa-solid fa-guitar"></i>
+          <h3>The Garage</h3>
+          <p>We're getting the band back together! The Garage is fully opened and ready to rock.</p>
+        </article>
+        <article class="elements">
+          <i class="fa-solid fa-star"></i>
+          <h3>Hall of Fame</h3>
+          <p>Inductee Signature Gallery, New Inductee exhibit. Featuring the Power of Rock Experience.</p>
+        </article>
+        <article class="elements">
+          <i class="fa-solid fa-map"></i>
+          <h3>THE WALL</h3>
+          <p>Stand beneath the Pink Floyd tour replica of The Wall, Berlin performance July 21, 1990.</p>
+        </article>
+        <article class="elements">
+          <i class="fa-solid fa-ranking-star"></i>
+          <h3>Legends of Rock</h3>
+          <p>A myriad of musical genres and highlights some of the world’s most iconic acts both past and present.</p>
+        </article>
+      </article>
+      <a href="#" class="programDetails">SEE THE WHOLE PROGRAM</a>
+      <button id="join" class="programBtn">Join ROCK & ROLL Hall of Fame Tour 2023</button>
+    </section>
+    <section id="artistSection">
+      <article id="featuredArtists"></div>
+    </section>
+  </main>
+  <footer class="more">
+    <div id="sponsor" class="partners">
+      <div class="holder">
+        <h2 class="title">Partner</h2>
+        <div class="underLine"></div>
+      </div>
+      <div>
+        <div class="partnersWrapper">
+          <img class="partner" src="images/partners/ohioArt.png" alt="Ohio Art">
+          <img class="partner" src="images/partners/medicalMutual.png" alt="Medical Mutual">
+          <img class="partner" src="images/partners/klipsch.png" alt="Klipsch">
+          <img class="partner" src="images/partners/unitedAirlines.png" alt="United Airlanes">
+          <img class="partner" src="images/partners/cuyahoga.png" alt="CuyaHoga">
+        </div>
+      </div>
+      <span></span>
+    </div>
+    <div id="rockhall" class="bottom">
+      <img class="footerLogo" src="images/logo.svg" alt="ROCK&ROLLHOF">
+      <div class="footerDesc">
+        <p>
+          <span>2023° Rock Hall.</span><span> Some rights reserved</span>
+        </p>
+        <span class="footerDetail"><br>Rock Hall is a registered 501(c)(3) non-profit in the United States.</span>
+      </div>
+    </div>
+  </footer>
+  <div class="seeMore">
+    <button class="seeMoreBtn">
+      <span>More</span>
+      <i class="fa-solid fa-angle-down"></i>
+    </button>
```

Comment:
- [optional] I think the `see more` button should only hide few of the `featured artist`; the `partner` and `footer` sections are hidden but only displayed when I clicked see more button. This is not a good user experience I think. What's your opinion? ❓ 

## Your App
![DSC_7491](https://user-images.githubusercontent.com/103386183/218041077-e92e2fdc-7d22-4496-8650-40ae74f8eff4.JPG)